tests/JobCreatorTest.php

```diff
@@ -135,58 +136,69 @@ public function provideGetInstallerVersion(): array
+            'tagfield5' => ['myaccount/silverstripe-tagfield', 'pulls/2.9/mybugfix', $currentMinorCms4],
+            'tagfield6' => ['myaccount/silverstripe-tagfield', 'pulls/burger/myfeature', $currentMinorCms4],
             // non-lockstepped repo, fallback to major version of installer (is missing 6.0 installer branch)
-            ['myaccount/silverstripe-tagfield', '4.0', '6.x-dev', [['name' => '6']], ['silverstripe/framework' => '^6']],
+            'tagfield7' => ['myaccount/silverstripe-tagfield', '4.0', '6.x-dev', [['name' => '6']], ['silverstripe/framework' => '^6']],
             // hardcoded repo version
-            ['myaccount/silverstripe-session-manager', '1', $nextMinorCms4],
-            ['myaccount/silverstripe-session-manager', '1.2', '4.10.x-dev'],
-            ['myaccount/silverstripe-session-manager', 'burger', $currentMinorCms4],
+            'session-manager1' => ['myaccount/silverstripe-session-manager', '1', $nextMinorCms4],
+            'session-manager2' => ['myaccount/silverstripe-session-manager', '1.2', '4.10.x-dev'],
+            'session-manager3' => ['myaccount/silverstripe-session-manager', 'burger', $currentMinorCms4],
             // hardcoded repo version using array
-            ['myaccount/silverstripe-html5', '2', $nextMinorCms4],
-            ['myaccount/silverstripe-html5', '2.2', '4.10.x-dev'],
-            ['myaccount/silverstripe-html5', '2.3', '4.10.x-dev'],
-            ['myaccount/silverstripe-html5', '2.4', '4.11.x-dev'],
-            ['myaccount/silverstripe-html5', 'burger', $currentMinorCms4],
+            'html1' => ['myaccount/silverstripe-html5', '2', $nextMinorCms4],
+            'html2' => ['myaccount/silverstripe-html5', '2.2', '4.10.x-dev'],
+            'html3' => ['myaccount/silverstripe-html5', '2.3', '4.10.x-dev'],
+            'html4' => ['myaccount/silverstripe-html5', '2.4', '4.11.x-dev'],
+            'html5' => ['myaccount/silverstripe-html5', 'burger', $currentMinorCms4],
             // force installer unlockedstepped repo
-            ['myaccount/silverstripe-serve', '2', $nextMinorCms4],
-            ['myaccount/silverstripe-behat-extension', '2', $nextMinorCms4],
+            'serve1' => ['myaccount/silverstripe-serve', '2', $nextMinorCms4],
+            'behat1' => ['myaccount/silverstripe-behat-extension', '2', $nextMinorCms4],
+            // repos that shouldn't have installer
+            'vendor-plugin1' => ['myaccount/vendor-plugin', '1', ''],
+            'vendor-plugin2' => ['myaccount/vendor-plugin', '2', ''],
+            'recipe-plugin1' => ['myaccount/recipe-plugin', '1', ''],
+            // random third-party repo
+            'random1' => ['myaccount/my-module', '4.0', '', [['name' => '6']], ['silverstripe/framework' => '^6']],
+            'random2' => ['myaccount/my-module', '4.0', '6.x-dev', [['name' => '6']], ['silverstripe/framework' => '^6'], 'silverstripe-module'],
```

Comment:
Since you're adding a type, should update the key to be random-type-module or something like that

Reply:
Done